README.md

```diff
@@ -16,7 +20,7 @@ in R based on Template Model Builder (TMB).
 
 You can install the current development version from GitHub with:
 
-```r
+``` r
```

Comment:
Don't know why these updates would be coming. I only knit `README.Rmd` which has many fewer updates. Any idea why this would happen?